Update Automation Util Contracts (#1080)
smartcontractkit/documentation: src/pages/chainlink-automation/util-overview.md

<YouTube id="https://www.youtube.com/watch?v=E7Rm1LUKhj4" />

This example teaches you how to create dynamic NFTs using Chainlink Automation.

> ### `VRFSubscriptionBalanceMonitor`

Reviewer: ```suggestion
### VRF Subscription Balance Monitor
```

[`VRFSubscriptionBalanceMonitor` Repository](https://github.com/smartcontractkit/chainlink/blob/vrf-subscription-balance-monitor/contracts/src/v0.8/upkeeps/VRFSubscriptionBalanceMonitor.sol)

> This contract automates the monitoring of your VRF subscription balance.

Reviewer: ```suggestion
This contract automates the monitoring of your VRF subscription balance to ensure there is sufficient funding for requests.
```

### Counting dNFT

> This repo houses an example that automates counting with a dynamic SVG. Refer to the repository for more instructions.

Reviewer: ```suggestion
This repository contains an example that automates counting with a dynamic SVG. Refer to the repository for more instructions.
```

[Counting dNFT Repository](https://github.com/smartcontractkit/smart-contract-examples/tree/main/counting-svg)

### Batch reveal Demo app

> Create batch-revealed NFT collections powered by Chainlink Automation & VRF [here](https://automation.chainlink-demo.app/).

Reviewer: ```suggestion
Create batch-revealed NFT collections powered by Chainlink Automation and VRF [here](https://automation.chainlink-demo.app/).
```
